src/pkg/services/m365/api/lists.go

```diff
@@ -2,12 +2,14 @@ package api
 
 import (
 	"context"
+	"strings"
 
 	"github.com/alcionai/clues"
 	"github.com/microsoftgraph/msgraph-sdk-go/models"
 	"github.com/microsoftgraph/msgraph-sdk-go/sites"
 
 	"github.com/alcionai/corso/src/internal/common/ptr"
+	betaAPI "github.com/alcionai/corso/src/internal/m365/service/sharepoint/api"
```

Comment:
is there any way to do this without the beta API?  I thought lists had caught up to the v1.0 API.

Reply:
@ryanfkeepers I do not think we are actually using beta API, the underlying functions (CloneListItem, CreateListFromBytes, ToListable) are not using [microsoft beta sdk](github.com/alcionai/corso/src/pkg/services/m365/api/graph/betasdk). Its just that the functions are present in that package.

Reply:
Okay, can you try to make this work without the beta package?